modules/lvms-about-lvmcluster-cr.adoc

```diff
@@ -0,0 +1,194 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-lvmcluster_{context}"]
+= About the LVMCluster custom resource
+
+After you have installed {lvms}, you must create an `LVMCluster` custom resource (CR) on a worker node.
+
+You can configure the `LVMCluster` CR to perform the following actions:
+
+* Create LVM volume groups that you can use to provision persistent volume claims (PVCs).
+* Configure a list of devices that you want to add to the LVM volume groups. 
+* Configure the requirements to select the nodes on which you want to create an LVM volume group, and the thin pool configuration for the volume group.
+* Force wipe the selected devices.
+
+You can create an `LVMCluster` CR using the OpenShift CLI (`oc`) or the {product-title} web console.
+
+include::snippets/lvms-creating-lvmcluster.adoc[]
+
+[discrete]
+== LVMCluster custom resource configuration
+
+The `LVMCluster` CR fields are described in the following table:
```

Comment:
I'd say tables plural and then just list them back-to-back. Because all of them technically make up the `LVMCluster` CR, not just this first one.

```suggestion
The `LVMCluster` CR fields are described in the following tables:
```

Reply:
Added all the fields in the same table. Hence, will keep it as is.